packages/gatsby-plugin-image/src/components/layout-wrapper.tsx

```diff
@@ -50,15 +47,17 @@ export function getSizer(
   width: number,
   height: number
 ): string {
-  let sizer: string | null = null
+  let sizer = ``
```

Comment:
So this was the hydration bugfix, right? Did you add tests to make sure we don't regress?

Reply:
I undid the tests that were skipped, I still need to update the test suite to actually test for flicker behavior